tools/utilities/pythonlibs/audio/training/make_featurizer.py

```diff
@@ -72,15 +72,15 @@ def _create_model(sample_rate, window_size, input_buffer_size, filterbank_type,
         last_node = builder.AddFFTNode(ell_model, ell.nodes.PortElements(last_node.GetOutputPort("output")))
 
     if not filterbank_nfft:
-        # default is to just use the output size of the fft node.
-        filterbank_nfft = last_node.GetOutputPort("output").Size()
+        # default is to just use the size of the fft.
+        filterbank_nfft = nfft
 
     if power_spec:
         fft_size = last_node.GetOutputPort("output").Size()
         square_node = builder.AddUnaryOperationNode(
             ell_model, ell.nodes.PortElements(last_node.GetOutputPort("output")),
             ell.nodes.UnaryOperationType.square)
-        denom = builder.AddConstantNode(ell_model, [fft_size], ell.nodes.PortType.smallReal)
+        denom = builder.AddConstantNode(ell_model, [nfft]*fft_size, ell.nodes.PortType.smallReal)
```

Comment:
great fix, thanks!

Reply:
Here as well "nfft" might not be set. I will make the changes and push it again.